README.rst

```diff
@@ -285,6 +285,18 @@ If you want to compile the applications under ``examples/``, you need
 to remove or rename the ``event.h`` from libev's installation, because
 it conflicts with libevent's installation.
 
+Notes for installation on Linux systems
+--------------------------------------------
+After installing nghttp2 tool suite with `make install` one might experience a similar error:
```

Comment:
This is reST document, and monospace text should be enclosed with double backtick like this:

```
``make install``
```